### PR TITLE
VTAdmin: Workflow status endpoint

### PR DESCRIPTION
**Pull request by beingnoble03**
## Description
Currently, VTAdmin misses out on `workflow status` endpoint. This endpoint is important for fetching/monitoring the progress of the workflow.

Example: http://localhost:14200/api/workflow_status/local/customer/commerce2customer
```json
{
  "result": {
    "shard_streams": {
      "customer/0": {
        "streams": [
          {
            "id": 1,
            "tablet": {
              "cell": "zone1",
              "uid": 201
            },
            "source_shard": "commerce/0",
            "position": "6d7be442-5964-11ef-8add-2ccc79fa7a75:1-54",
            "status": "Running",
            "info": "VStream Lag: 0s"
          }
        ]
      }
    },
    "traffic_state": "Reads Not Switched. Writes Not Switched"
  },
  "ok": true
}
```

## Related Issue(s)
- #16537 

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [ ] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on CI?
-   [ ] Documentation was added or is not required

### CODE REVIEW

File: go/vt/vtadmin/api.go [hunk 1]
```diff
@@ -417,6 +417,7 @@ func (api *API) Handler() http.Handler {
 	router.HandleFunc("/vtexplain", httpAPI.Adapt(vtadminhttp.VTExplain)).Name("API.VTExplain")
 	router.HandleFunc("/workflow/{cluster_id}/{keyspace}/{name}", httpAPI.Adapt(vtadminhttp.GetWorkflow)).Name("API.GetWorkflow")
 	router.HandleFunc("/workflows", httpAPI.Adapt(vtadminhttp.GetWorkflows)).Name("API.GetWorkflows")
+	router.HandleFunc("/workflow_status/{cluster_id}/{keyspace}/{name}", httpAPI.Adapt(vtadminhttp.GetWorkflowStatus)).Name("API.GetWorkflowStatus")
```
Comment (rohit-nayak-ps) [inline go/vt/vtadmin/api.go:420]: We should probably do this like 
`/workflow/{cluster_id}/{keyspace}/{name}/status` similar to the keyspace APIs, for example:
```
        router.HandleFunc("/keyspace/{cluster_id}", httpAPI.Adapt(vtadminhttp.CreateKeyspace)).Name("API.CreateKeyspace").Methods("POST")
	router.HandleFunc("/keyspace/{cluster_id}/{name}", httpAPI.Adapt(vtadminhttp.DeleteKeyspace)).Name("API.DeleteKeyspace").Methods("DELETE")
	router.HandleFunc("/keyspace/{cluster_id}/{name}", httpAPI.Adapt(vtadminhttp.GetKeyspace)).Name("API.GetKeyspace")
	router.HandleFunc("/keyspace/{cluster_id}/{name}/rebuild_keyspace_graph", httpAPI.Adapt(vtadminhttp.RebuildKeyspaceGraph)).Name("API.RebuildKeyspaceGraph").Methods("PUT", "OPTIONS")
	router.HandleFunc("/keyspace/{cluster_id}/{name}/remove_keyspace_cell", httpAPI.Adapt(vtadminhttp.RemoveKeyspaceCell)).Name("API.RemoveKeyspaceCell").Methods("PUT", "OPTIONS")
	router.HandleFunc("/keyspace/{cluster_id}/{name}/validate", httpAPI.Adapt(vtadminhttp.ValidateKeyspace)).Name("API.ValidateKeyspace").Methods("PUT", "OPTIONS")
	router.HandleFunc("/keyspace/{cluster_id}/{name}/validate/schema", httpAPI.Adapt(vtadminhttp.ValidateSchemaKeyspace)).Name("API.ValidateSchemaKeyspace").Methods("PUT", "OPTIONS")
	router.HandleFunc("/keyspace/{cluster_id}/{name}/validate/version", httpAPI.Adapt(vtadminhttp.ValidateVersionKeyspace)).Name("API.ValidateVersionKeyspace").Methods("PUT", "OPTIONS")
```

**Review by rohit-nayak-ps — COMMENTED** [review]
Looks good. I just had one comment about the API endpoint. 

I tested the current endpoint locally and added the url/output to the description.

File: go/vt/vtadmin/api.go (hunk 1 above)
Comment (beingnoble03) [reply go/vt/vtadmin/api.go]: Ah, makes sense! done.

**Review by rohit-nayak-ps — APPROVED** [review]
Nice work.

**Review by notfelineit — APPROVED** [review]
LGTM. So excited to see more workflow stuff in VTAdmin 😄

**Pull request merged**